scripts/gen_handles.py

```diff
@@ -65,10 +62,17 @@ def parse_args():
 
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="Print extra debugging information")
+
+    parser.add_argument("-z", "--zephyr-base", required=True,
```

Comment:
This should not be required.  The script will work fine run manually if the retrieval of a default value from the environment is not removed.  Running the script manually is useful, and having to supply the zephyr base explicitly is annoying.

Reply:
correct, if the script is intended to often be run directly from command line, then the possibility of providing it in environment is useful. 
But in such a case, a proper error message should be provided if it fails, and not a stack trace.

Regarding `twister`, then `twister` calls CMake which will have an internal `ZEPHYR_BASE` variable, which will be fetched from environment on initial invocation.

Otherwise, changing the ZEPHYR_BASE between ninja invocation in same build folder will cause some scripts to be out of sync with other parts of the build system.

Reply:
Updated, and now provides a nice error message if `-z` / `--zephyr-base` is not given and not available as `ZEPHYR_BASE` in environment.